notebooks/json_blocks/WET/BLOCK-T413.ipynb

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.            name="maintel/set_dof.py",
</pre><p>Setting up <code>set_dof.py</code> like this requires a rewrite of <code>set_dof.py</code> right? Is that work recorded as a TODO somewhere and linked to this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>